src/car/distance/DistanceCar.cpp

```diff
@@ -1,5 +1,5 @@
-#include <math.h>   // NOLINT(modernize-deprecated-headers)
-#include <stdint.h> // NOLINT(modernize-deprecated-headers)
+#include <math.h>
+#include <stdint.h>
```

Comment:
:warning: **modernize-deprecated-headers** :warning:
inclusion of deprecated C++ header 'stdint.h'; consider using 'cstdint' instead
```suggestion
#include <cstdint>```